src/Extensions/ExtHostClient.rei

```diff
@@ -55,11 +159,12 @@ let start:
   ) =>
   t;
 let activateByEvent: (string, t) => unit;
-let executeContributedCommand: (string, t) => unit;
+let executeContributedCommand:
+  (~arguments: list(Core.Json.t)=?, string, t) => unit;
 let acceptWorkspaceData: (Workspace.t, t) => unit;
 let addDocument: (Protocol.ModelAddedDelta.t, t) => unit;
 let updateDocument:
-  (Core.Uri.t, Protocol.ModelChangedEvent.t, bool, t) => unit;
+  (Core.Uri.t, Protocol.ModelChangedEvent.t, ~dirty: bool, t) => unit;
```

Comment:
Labelled the `~dirty` argument, since `bool` alone is almost void of meaning and should pretty much never be used as a positional function argument. Either label it or use a variant type of some kind that says what it actually does.

Reply:
Good point. Without a label you have to dive into the implementation to figure it out. Thanks for fixing this.